dicttoxml.py

```diff
@@ -254,13 +257,16 @@ def convert_dict(obj, ids, parent, attr_type, item_func, cdata):
     return ''.join(output)
 
 
-def convert_list(items, ids, parent, attr_type, item_func, cdata):
+def convert_list(items, ids, parent, attr_type, item_func, cdata, fold_list):
     """Converts a list into an XML string."""
     LOG.info('Inside convert_list()')
     output = []
     addline = output.append
 
-    item_name = item_func(parent)
+    if fold_list:
+        item_name = item_func(parent)
+    else:
+        item_name = parent
```

Comment:
Maybe you could explicitly fail-fast if `item_func` is explicitly input into dicttoxml API **and** `fold_list` is also explicitly defined to be False? Might be preferable instead of silently ignoring and causing a surprise.